content/en/docs/setup/install/multicluster/gateways/index.md

```diff
@@ -542,7 +542,7 @@ Uninstall Istio by running the following commands on **every cluster**:
 
 {{< text bash >}}
 $ istioctl manifest generate \
-    -f install/kubernetes/operator/examples/multicluster/values-istio-multicluster-gateways.yaml \
+    -f manifests/examples/multicluster/values-istio-multicluster-gateways.yaml \
```

Comment:
Is this update correct from the downloadable archive perspective?

Reply:
I am using the downloaded archive for 1.6-alpha.2